packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -471,6 +471,7 @@ describe("calcite-list", () => {
 
     it("works using a mouse", async () => {
       const page = await createSimpleList();
+      //const listOrderChangeSpy = await page.spyOnEvent("calciteListOrderChange");
```

Comment:
@jcfranco I need your help with this one. If I add this spy, the test fails with the following...

```
ReferenceError: You are trying to `import` a file after the Jest environment has been torn down. From src/components/list/list.e2e.ts.

      at Object.get (../../node_modules/@stencil/core/testing/index.js:7:14)
      at Object.<anonymous> (../../node_modules/@stencil/core/testing/index.js:3772:79)

 RUNS  src/components/list/list.e2e.ts
/Users/dris0000/Sites/git/calcite-design-system/node_modules/@stencil/core/testing/index.js:3925
    throw new Error("The page was already closed");
          ^

Error: Caught error after test environment was torn down
```

Reply:
Took a look and it looks like this error is caused by the drag event payload being serialized and there being circular JSON structures from the payload elements. I don't know yet what the proper fix for this would be, but you can work around it by manually adding an event listener to the page and tracking/storing the times it was emitted changes on `window` and then using that value to assert the final call count.

Reply:
Cool yeah that works for now. Thanks 👍